web/packages/teleport/src/Account/Account.tsx

```diff
@@ -156,122 +152,76 @@ export function Account({
     }
   }
 
-  function onPasswordChange() {
-    addNotification('info', 'Your password has been changed.');
-    onPasswordChangeCb();
-  }
-
-  function onAddDeviceSuccess() {
-    const message =
-      newDeviceUsage === 'passwordless'
-        ? 'Passkey successfully saved.'
-        : 'MFA method successfully saved.';
-    addNotification('info', message);
-    onDeviceAdded();
-  }
-
-  function onDeleteDeviceSuccess() {
-    const message =
-      deviceToRemove.usage === 'passwordless'
-        ? 'Passkey successfully deleted.'
-        : 'MFA method successfully deleted.';
-    addNotification('info', message);
-    onDeviceRemoved();
-  }
-
   return (
     <Relative>
-      <FeatureBox maxWidth={1440} margin="auto">
+      <FeatureBox margin="auto">
```

Comment:
why did we move the maxWidth from here to some flex component down there?

Reply:
From the [designs](https://www.figma.com/design/rlcaLyLb3AkG6MW4fnRSAG/Profile-Settings?node-id=188-14685&t=3kTEyhMP4wpXZtQi-0), it looked to me that the designer wanted the settings component to take up the same amount of space at its max width with the side nav attached to its side.

Happy to move this attribute back, if you think otherwise?

Reply:
ah no, that makes much more sense. ok thanks!